tests/functional/conftest.py

```diff
@@ -5,7 +5,6 @@
 import pytest
 
 from firefox_puppeteer import Puppeteer
-from firefox_puppeteer.ui.browser.window import BrowserWindow
```

Comment:
Please return that line. Because of the way Puppeteer works, that line is necessary though you don't even use it. It causes some initialization stuff to happen in the background. We may be investigating how to move from Puppeteer to selenium later on to not have to deal with such issues, but for now, that has to remain.